index.html

```diff
@@ -93,15 +93,16 @@
 
   }, 2000);
 
-  // Update modified files on left file panel every X seconds
+  // group all monitors here, call every 3 seconds
```

Comment:
Is 3 seconds chosen arbitrarily as a value?

Reply:
Original VisualGit chose 3 seconds (3000 ms) to refresh their staged and unstaged file list. I then put my branch/tag live-reload and commit graph change detection into same place.